browser/brave_ads/tooltips/BUILD.gn

```diff
@@ -24,4 +24,8 @@ source_set("tooltips") {
     "//chrome/browser/profiles:profile",
     "//ui/base",
   ]
+
+  if (is_win || is_linux || is_mac) {
```

Comment:
Adding view deps to `brave/browser/ui/BUILD.gn` would be much better like this.
```
if (brave_adaptive_captcha_enabled && toolkit_views) {
  deps += [ "//brave/browser/ui/views/brave_tooltips" ]
}
```

Reply:
`brave_adaptive_captcha` is already asserted in this file, so I don't think we need to add it there, but I did remove the `is_win || is_linux || is_mac` and added `toolkit_views` - thanks!